packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -530,3 +544,39 @@ class _TooltipOverlay extends StatelessWidget {
     );
   }
 }
+
+class _TooltipSemanticsWidget extends SingleChildRenderObjectWidget {
+  const _TooltipSemanticsWidget({
+    Key key,
+    Widget child
+  }) : super(key: key, child: child);
+
+  @override
+  RenderObject createRenderObject(BuildContext context) {
+    return _RenderTooltipSemanticsWidget();
+  }
+}
+
+class _RenderTooltipSemanticsWidget extends RenderProxyBox {
+  _RenderTooltipSemanticsWidget({RenderBox child}) : super(child) {
+    markNeedsSemanticsUpdate();
+  }
+
+  String label;
+
+  @override
+  void assembleSemanticsNode(
```

Comment:
`assembleSemanticsNode` only runs if semantics are turned on. Semantics are only turned on if an a11y service (e.g. TalkBack or VoiceOver) is running. I thought this label also needs to show if semantics are turned off? Based on that, I would expect that the tooltip is not showing a message when a11y services are turned off?

Reply:
Side-note: For technical reasons, a11y are always turned on when running in a simulator. Did you try the tooltip thing on a real device without having any a11y services (outside of scaling the text) enabled? You may have to force-quit and restart the app after turning the a11y service off to delete the cached `label`.